# HG changeset patch
# User Rik <rik@octave.org>
# Date 1363027198 25200
#      Mon Mar 11 11:39:58 2013 -0700
# Node ID c8bf749fe6f88d054356add815cfc8c8563fe4bd
# Parent  6601c66280c45ba1f7099206267c116d1842d1cb
oct-parse.in.yy: Remove unused variable causing compilation warning.

* libinterp/parse-tree/oct-parse.in.yy(parse_fcn_file): Remove unused
"bool eof;" line.

diff --git a/libinterp/parse-tree/oct-parse.in.yy b/libinterp/parse-tree/oct-parse.in.yy
--- a/libinterp/parse-tree/oct-parse.in.yy
+++ b/libinterp/parse-tree/oct-parse.in.yy
@@ -3176,18 +3176,16 @@ parse_fcn_file (const std::string& full_
 
   if (! full_file.empty ())
     ffile = gnulib::fopen (full_file.c_str (), "rb");
 
   frame.add_fcn (safe_fclose, ffile);
 
   if (ffile)
     {
-      bool eof;
-
       // octave_parser constructor sets this for us.
       frame.protect_var (CURR_LEXER);
 
       octave_parser curr_parser (ffile);
 
       curr_parser.curr_class_name = dispatch_type;
       curr_parser.autoloading = autoload;
       curr_parser.fcn_file_from_relative_lookup = relative_lookup;
